x/ibc/07-tendermint/client/cli/tx.go

```diff
@@ -67,7 +67,7 @@ func GetCmdCreateClient(cdc *codec.Codec) *cobra.Command {
 			if lvl == "default" {
 				trustLevel = lite.DefaultTrustLevel
 			} else {
-				trustLevel, err = parseFraction(args[2])
+				trustLevel, err = parseFraction(lvl)
```

Comment:
ops

Reply:
The args below all need to be bumped down by 1 index